Update README for ALOHA 2024 (#246)
astropy/astropy-workshop: README.md

Reviewer: Should we link specific topic to the notebook(s) that will be used, if applicable?

Author: I think we will will need to wait to do that until after the reorg....

# Astropy Learning at Observatories of HAwaii (ALOHA)

*Workshops at Subaru and Keck*

## PRE-WORKSHOP TO-DOs

> ### Install Python if you do not have it

Reviewer: I cannot get to L16 but I do not think it is accurate for this workshop?

`If you have any trouble, we will have facilitators on-site as early as 8:30 AM local time who can help you in person.`

Author: Correct, that is not accurate, fixing

# Astropy Learning at Observatories of HAwaii (ALOHA)

*Workshops at Subaru and Keck*

## PRE-WORKSHOP TO-DOs

### Install Python if you do not have it

Please be sure your laptop is properly configured before the workshop by following the
[installation and setup instructions](00-Install_and_Setup).

*Warning*: Installation and setup could take as long as *one hour* depending on your current configuration and internet speeds.

> DO NOT WAIT UNTIL THE DAY OF THE WORKSHOP.s

Reviewer: ```suggestion
DO NOT WAIT UNTIL THE DAY OF THE WORKSHOP.
```

### Review some introductory Python if you have not used Python

That material is in the folder `02-PythonIntro`. A link to that -- so that you can read it even if you have not used Python before -- is [here](https://github.com/mwcraig/astropy-workshop/tree/ALOHA-2024/02-PythonIntro). Click on the individual notebooks, whose names end `.ipynb`, to open them.

Doing this before the workshop will put you in a better position to follow along with the workshop materials.

## Schedules

At both sets of workshops there will be science use cases integrated into
the presentation topics.

Note that the materials are very similar between the two workshops, although the emphasis on the Subaru and Keck days are slightly different.

> ### Subaru, Dec 2 and Dec 3, 2024

Reviewer: ```suggestion
### Subaru, Dec 2-3, 2024
```

> #### Day 1: Mon, December 2, 2024

Reviewer: ```suggestion
#### Day 1: Mon, December 2, 2024 (Subaru workshop day 1)
```

* 10:00 AM - 10:30 AM: Intro to astropy and orientation to the astropy ecosystem
* 10:30 AM - 11:00 AM: More astropy fundamentals and science examples by Matt Craig
* 11:00 AM - 11:30 PM: Visualization options with astropy and Python by Erik Tollerud
* 11:30 AM - 12:00 PM: Visualization Q&A / User stories
* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: Photometry with Astropy by Pey Lian Lim
* 02:30 PM - 04:30 PM: Office hours / "code along"
* $\color{gray}{Organizers' \space BBQ \space (private \space event)}$

> ### Day 2: Tue, December 3, 2024 Subaru workshop day 2:

Reviewer: ```suggestion
#### Day 2: Tue, December 3, 2024 (Subaru workshop day 2)
```

* 10:00 AM - 10:30 AM: Spectroscopy with Astropy by Erik Tollerud
* 10:30 AM - 11:00 AM: Spectroscopy continued / Q&A
* 11:00 AM - 11:30 PM: DevOps Q&A by Pey Lian Lim
* 11:30 AM - 12:00 PM: More astropy science use cases by Matt Craig
* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: Deeper dive (a couple of topics based on participants interest/needs)
* 02:30 PM - 04:30 PM: Office hours / "code along"
* $\color{gray}{Organizers' \space picnic \space and \space observation \space shadowing \space (private \space event)}$

> ### Day 3: Wed, December 4, 2024 Keck workshop day 1:

Reviewer: ```suggestion
### Keck, Dec 4-5, 2024

#### Day 3: Wed, December 4, 2024 (Keck workshop day 1)
```

* 10:00 AM - 10:30 AM: Intro to astropy package and orientation to the Astropy ecosystem by Matt Craig
* 10:30 AM - 11:00 AM: More astropy fundamentals and science examples by Matt Craig
* 11:00 AM - 11:30 AM: Visualization options with astropy and Python by Erik Tollerud
* 11:30 AM - 12:00 PM: Visualization Q&A / User stories
* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: DevOps Q&A by Pey Lian Lim / discussion
* 02:30 PM - 04:30 PM: Office hours / "code along"

> ### Day 4: Thu, December 5, 2024 Keck workshop day 2:

Reviewer: ```suggestion
#### Day 4: Thu, December 5, 2024 (Keck workshop day 2)
```

As an alternative, a workshop session can be run on mybinder.org via this link: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/astropy/astropy-workshop/HEAD)

### Review some introductory Python if you have not used Python

That material is in the folder `02-PythonIntro`. A link to that -- so that you can read it even if you have not used Python before -- is [here](https://github.com/mwcraig/astropy-workshop/tree/ALOHA-2024/02-PythonIntro). Click on the individual notebooks, whose names end `.ipynb`, to open them.

Doing this before the workshop will put you in a better position to follow along with the workshop materials.

## Schedules

At both sets of workshops there will be science use cases integrated into
the presentation topics.

> 

Reviewer 2: ```suggestion

Note that the materials are very similar between the two workshops, although the emphasis on the Subaru and Keck days are slightly different.

```

Reviewer: Also remove the redundant "this is a duplicate" statement in individual entries.

* 10:00 AM - 10:30 AM: Intro to astropy and orientation to the astropy ecosystem
* 10:30 AM - 11:00 AM: More astropy fundamentals and science examples
* 11:00 AM - 11:30 PM: Visualization options with astropy and Python by Erik Tollerud (tentative)
* 11:30 AM - 12:00 PM: Visualization Q&A / User stories
* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: Photometry with Astropy by Pey Lian Lim (tentative)
* 02:30 PM - 04:30 PM: Office hours / "code along"
* $\color{gray}{Organizers' \space BBQ \space (private \space event)}$

### Day 2: Tue, December 3, 2024 Subaru workshop day 2:

* 10:00 AM - 10:30 AM: Spectroscopy with Astropy by Erik Tollerud
* 10:30 AM - 11:00 AM: Spectroscopy continued / Q&A
* 11:00 AM - 11:30 PM: DevOps Q&A by Pey Lian Lim
* 11:30 AM - 12:00 PM: More astropy science use cases by Matt Craig (tentative)
* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: Deeper dive (a couple of topics based on participants interest/needs)
* 02:30 PM - 04:30 PM: Office hours / "code along"
* $\color{gray}{Organizers' \space picnic \space and \space observation \space shadowing \space (private \space event)}$

### Day 3: Wed, December 4, 2024 Keck workshop day 1:

> * 10:00 AM - 10:30 AM: Intro to astropy and orientation to the astropy ecosystem by Pey Lian Lim (tentative) (this is a repeat of Day 1 opening talk)
> * 10:30 AM - 11:00 AM: More astropy fundamentals and science examples by Matt Craig (this is a repeat of Day 1)

Reviewer 2: ```suggestion
* 10:00 AM - 10:30 AM: Intro to astropy and orientation to the astropy ecosystem by Pey Lian Lim (tentative) 
* 10:30 AM - 11:00 AM: More astropy fundamentals and science examples by Matt Craig 
```

> As an alternative, a workshop session can be run on mybinder.org via this link: [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/astropy/astropy-workshop/HEAD)

Reviewer: Does the Binder thingy apply for this workshop? If not, we should remove.

Author: binder should be a very last-ditch thing because it takes much longer to get going than it used to. We would also need to update it for this branch, so I'll take it out for now

### Review some introductory Python if you have not used Python

That material is in the folder `02-PythonIntro`. A link to that -- so that you can read it even if you have not used Python before -- is [here](https://github.com/mwcraig/astropy-workshop/tree/ALOHA-2024/02-PythonIntro). Click on the individual notebooks, whose names end `.ipynb`, to open them.

Doing this before the workshop will put you in a better position to follow along with the workshop materials.

## Schedules

At both sets of workshops there will be science use cases integrated into
the presentation topics.

Note that the materials are very similar between the two workshops, although the emphasis on the Subaru and Keck days are slightly different.

### Subaru, Dec 2 and Dec 3, 2024

#### Day 1: Mon, December 2, 2024

> * 10:00 AM - 10:30 AM: Intro to astropy and orientation to the astropy ecosystem

Reviewer: ```suggestion
* 10:00 AM - 10:30 AM: Intro to astropy package and orientation to the Astropy ecosystem by Matt Craig
```

> * 10:30 AM - 11:00 AM: More astropy fundamentals and science examples

Reviewer: ```suggestion
* 10:30 AM - 11:00 AM: More astropy fundamentals and science examples by Matt Craig
```

> * 11:00 AM - 11:30 PM: Visualization options with astropy and Python by Erik Tollerud (tentative)

Reviewer: ```suggestion
* 11:00 AM - 11:30 PM: Visualization options with astropy and Python by Erik Tollerud
```

* 11:30 AM - 12:00 PM: Visualization Q&A / User stories
* 12:00 PM - 01:30 PM: Lunch

> * 01:30 PM - 02:30 PM: Photometry with Astropy by Pey Lian Lim (tentative)

Reviewer: ```suggestion
* 01:30 PM - 02:30 PM: Photometry with Astropy by Pey Lian Lim
```

* 02:30 PM - 04:30 PM: Office hours / "code along"
* $\color{gray}{Organizers' \space BBQ \space (private \space event)}$

### Day 2: Tue, December 3, 2024 Subaru workshop day 2:

* 10:00 AM - 10:30 AM: Spectroscopy with Astropy by Erik Tollerud
* 10:30 AM - 11:00 AM: Spectroscopy continued / Q&A
* 11:00 AM - 11:30 PM: DevOps Q&A by Pey Lian Lim

> * 11:30 AM - 12:00 PM: More astropy science use cases by Matt Craig (tentative)

Reviewer: ```suggestion
* 11:30 AM - 12:00 PM: More astropy science use cases by Matt Craig
```

* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: Deeper dive (a couple of topics based on participants interest/needs)
* 02:30 PM - 04:30 PM: Office hours / "code along"
* $\color{gray}{Organizers' \space picnic \space and \space observation \space shadowing \space (private \space event)}$

### Day 3: Wed, December 4, 2024 Keck workshop day 1:

> * 10:00 AM - 10:30 AM: Intro to astropy and orientation to the astropy ecosystem by Pey Lian Lim (tentative) 

Reviewer: ```suggestion
* 10:00 AM - 10:30 AM: Intro to astropy package and orientation to the Astropy ecosystem by Matt Craig
```

* 10:30 AM - 11:00 AM: More astropy fundamentals and science examples by Matt Craig 

> * 11:00 AM - 11:30 AM: Visualization options with astropy and Python by Erik Tollerud (tentative)

Reviewer: ```suggestion
* 11:00 AM - 11:30 AM: Visualization options with astropy and Python by Erik Tollerud
```

* 11:30 AM - 12:00 PM: Visualization Q&A / User stories
* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: DevOps Q&A by Pey Lian Lim / discussion
* 02:30 PM - 04:30 PM: Office hours / "code along"

### Day 4: Thu, December 5, 2024 Keck workshop day 2:

> * 10:00 AM - 10:30 AM: Photometry with Astropy by Pey Lian Lim (tentative)

Reviewer: ```suggestion

* 10:00 AM - 10:30 AM: Photometry with Astropy by Pey Lian Lim
```

* 10:30 AM - 11:00 AM: Photometry continued / Q&A
* 11:00 AM - 11:30 AM: Spectroscopy with Astropy by Erik Tollerud

> * 11:30 AM - 12:00 PM: More astropy science use cases by Matt Craig (tentative)

Reviewer: ```suggestion
* 11:30 AM - 12:00 PM: More astropy science use cases by Matt Craig
```

* 12:00 PM - 01:30 PM: Lunch
* 01:30 PM - 02:30 PM: Deeper dive (topic based on participants interest)
* 02:30 PM - 04:30 PM: Office hours / "code along"
* Group dinner
* 07:00 PM - 09:00 PM: Astronomy On Tap at Hilo Town Tavern in Hilo, by Matt Craig

> 

Reviewer: ```suggestion
```
